### PR TITLE
Add repos, guides, update README

### PR DESCRIPTION
**Pull request by hieplpvip**
**What has been done in this PR ?**
- Add guide for audio and trackpad
- Add Asus Vivobook S510UA
- Add Lenovo U330/U430/U530
- Add Lenovo Y50-70
- Add HP ProBook/EliteBook/ZBook
- Add HP Envy J/K/Q/N series
- Update README

### CODE REVIEW

File: Post/Audio/Get audio working in Hackintosh.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* EMlyDinEsH: [Complete AppleHDA Patching Guide](http://forum.osxlatitude.com/index.php?/topic/1946-complete-applehda-patching-guide/)
+* Mirone: [Guide to patch AppleHDA for your codec](http://www.insanelymac.com/forum/topic/295001-guide-to-patch-applehda-for-your-codec/)
+* insanelydeepak: [AIO Complete Guide to Patch AppleHDA for Your Codec](http://osxarena.com/2015/03/best-all-in-one-patch-applehda-guide/)
+
+### How to install AppleALC
+1. Download AppleALC from Release page, and install it.
+2. In DSDT, rename HDAS (or AZAL) to HDEF and B0D3 to HDAU.
+This can be done by patching DSDT or using Clover's patching feature: add these [patches](https://github.com/RehabMan/OS-X-Clover-Laptop-Config/blob/master/hotpatch/config.plist#L156-L179) to `config.plist/ACPI/DSDT/Patches`
+3. Inject custom layout-id for your codec in `config.plist/Devices/Audio/Inject` like this:
+```
```
Comment (huangyz0918) [inline Post/Audio/Get audio working in Hackintosh.md]: Added __```xml__ label in the code block will be better.

File: Post/Trackpad/Install kexts for Trackpad.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Install kexts for Trackpad
+
+All trackpads are not supported natively in macOS. You have to manually install kexts to get it working.
+
+There are two protocols used to communicate with trackpad:
+* PS/2 (old)
```
Comment (huangyz0918) [inline Post/Trackpad/Install kexts for Trackpad.md]: Really good 👍

File: Post/Audio/Get audio working in Hackintosh.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Get audio working in Hackintosh
```
Comment (huangyz0918) [inline Post/Audio/Get audio working in Hackintosh.md]: how about rename this file as `README.md`? So it can be shown in a more obviously way.

**Comment by huangyz0918** [comment]
@hieplpvip Thank you very much for your PR! I have some tiny improvements, and I think we can merge it after you updated that.

**Comment by hieplpvip** [comment]
I want to write how to create usb installer, where should I put it?

**Comment by huangyz0918** [comment]
@hieplpvip Great! You can put it here: [quick start](https://github.com/huangyz0918/Hackintosh-Installer-University/blob/master/quick/quick.md).

**Pull request merged**